targets/targets.json

```diff
@@ -2740,6 +2740,58 @@
         ],
         "device_name": "STM32G071RBTx",
         "bootloader_supported": true
+    },
+	"NUCLEO_G031K8": {
+        "inherits": [
+            "MCU_STM32_BAREMETAL"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "MORPHO"
+        ],
+        "core": "Cortex-M0+",
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
+                "value": "USE_PLL_HSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
```

Comment:
Maybe i'm wrong but i can't see any pins on this package for `HSE` or `EXTC` there is only `OSC32_IN/OUT`

Reply:
Yes, this chip does not have HSE xtal inputs. Corrected by c3a59a253b48d54c3ce383f87c87d0ba42ad910d